<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buys</class>
 <widget class="QWidget" name="Buys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="BuysGB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Buys</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BuysSearch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>           Search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Search by</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>271</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Bill number</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Product ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Client ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>    Client ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ClientIDCB"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>   Product ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ProductIDCB"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
